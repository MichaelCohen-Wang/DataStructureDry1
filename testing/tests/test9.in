add_ship 390 28
add_pirate 576 390 18
get_cannons 390
add_pirate 366 390 25
get_cannons 390
get_treasure 366
add_pirate 188 390 25
get_cannons 390