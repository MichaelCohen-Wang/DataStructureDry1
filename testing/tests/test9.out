add_ship: SUCCESS
add_pirate: SUCCESS
get_cannons: SUCCESS, 28
add_pirate: SUCCESS
get_cannons: SUCCESS, 28
get_treasure: SUCCESS, 25
add_pirate: SUCCESS
get_cannons: SUCCESS, 28
